### PR TITLE
add err handling in gce/gci

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

add error handling

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/gci/configure_helper_test.go [hunk 1]
```diff
@@ -148,7 +148,10 @@ func (c *ManifestTestCase) mustLoadPodFromManifest() {
 }
 
 func (c *ManifestTestCase) tearDown() {
-	os.RemoveAll(c.kubeHome)
+	err := os.RemoveAll(c.kubeHome)
+	if err != nil {
```
Comment (yguo0905) [inline cluster/gce/gci/configure_helper_test.go]: optional:
`if err := os.RemoveAll(c.kubeHome); err != nil {`
...

Comment (hwdef) [reply cluster/gce/gci/configure_helper_test.go]: Thanks for you suggestion, I changed it.
Cloud you please add a lgtm label for this PR again?

**Pull request merged**